src/mapkit.cs

```diff
@@ -32,6 +32,7 @@
 using UIView=AppKit.NSView;
 using UIEdgeInsets=AppKit.NSEdgeInsets;
 using UIColor=AppKit.NSColor;
+using UIScene=AppKit.NSColor;
```

Comment:
AppKit doesn't have an equivalent to `UIScene`, maybe it would be better to use `NSObject` here?